apps/frontend/src/components/TimeCardPage/Evaluations.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { Button } from "@chakra-ui/react";
+import { EditIcon } from "@chakra-ui/icons";
+import { EvaluationModal } from "./EvaluationModal";
+import { CommentSchema } from "src/schemas/RowSchema";
+
+export function Evaluations({
+  commentList,
+  openEvaluationForm,
+  updateCommentList,
+}) {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [commentToEdit, setCommentToEdit] = useState<CommentSchema | null>(
+    null
+  );
+
+  const handleModalOpen = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleModalClose = () => {
+    setIsModalOpen(false);
+  };
+
+  const handleCommentDelete = (commentToDelete: CommentSchema) => {
+    const updatedComments = commentList.filter(
+      (comment) => comment !== commentToDelete
+    );
+    updateCommentList(updatedComments);
+  };
+
+  const handleCommentSubmit = (comment: CommentSchema) => {
+    if (commentToEdit) {
+      const index = commentList.findIndex((c) => c === commentToEdit);
+      const updatedCommentList = [...commentList];
+      updatedCommentList[index] = comment;
+      updateCommentList(updatedCommentList);
+
+      setIsModalOpen(false);
+    } else {
+      const updatedCommentList = [...commentList, comment];
+      updateCommentList(updatedCommentList);
```

Comment:
Nice use of callback functions!

We might want to look at changing this in the future to instead just call something like a `saveTimesheet` function so that this actually saves to the `apiClient`, so maybe leave a TODO comment here or in the `Timesheet` file that we'll need to change it to hook it up to the backend.